tutorials/cloud-iot-gateways-rpi/index.md

```diff
@@ -0,0 +1,272 @@
+
+## Objectives
+
+- Create a new gateway and bind devices to it
+- Demonstrate changing configurations on a device bound to a gateway
+- Demonstrate sending telemetry from a bound device to Cloud IoT Core
+
+## Before you begin
+
+This tutorial assumes you have a Google Cloud Platform (GCP) account and have completed the setup steps outlined in the [Cloud IoT Core getting started guide][iot-start]. For quick cleanup, create a new GCP project to use just for this tutorial.
+
+For more information about the different authentication methods Cloud IoT Core offers, [see Manage Gateways][iot-gateways].
+
+[iot-start]: https://cloud.google.com/iot/docs/how-tos/getting-started
+[iot-gateways]: https://cloud.google.com/iot/docs/how-tos/gateways/manage-gateways
+
+## Costs
+
+This tutorial uses billable components of Cloud Platform, including:
+
+- Cloud IoT Core
+- Cloud Pub/Sub
+
+This tutorial should not generate any usage that would not be covered by the free tier, but you can use the [pricing calculator](https://cloud.google.com/products/calculator/#id=411d8ca1-210f-4f2c-babd-34c6af2b5538) to generate a cost estimate based on your projected usage.
+
+## Required Hardware
+
+- [Raspberry Pi 3][rpi] Model B (Other models should work, but has not been verified)
+- MicroSD card for [Raspbian](https://www.raspberrypi.org/downloads/raspbian/) (8GB+ recommended)
+- MicroSD card reader
+- USB keyboard
+- MicroUSB to USB-A cable
+- Monitor with HDMI input
+- HDMI cable
+- LED
+- [Adafruit DHT22 Temperature/Humidity Sensor](https://www.adafruit.com/product/385)
+- 10k Ohm resistor
+- Breadboard and jumper wires
+- Wires
+
+## Enable Cloud IoT Core and Cloud Pub/Sub APIs
+
+Click **Enable API** for each of the following:
+
+- [Cloud IoT Core][cloud-iot] (for managing gateways and devices)
+- [Cloud Pub/Sub](http://console.developers.google.com/cloudpubsub) (for ingesting device telemetry)
+
+[cloud-iot]: https://console.developers.google.com/iot
+
+## Create a Registry
+
+First, create a device registry that will contain your gateway and devices.
+
+1. Open the [Cloud IoT Core console][cloud-iot].
+2. Ensure the right project is selected in the upper left.
+3. Click on **Create a device registry**.
+4. For **Registry ID**, enter `my-registry`.
+5. Select the region closest to you, but for the purposes of this demo we'll use `us-central1`.
+6. Ensure the MQTT protocol is enabled for this registry.
+7. Under **Cloud Pub/Sub topics**, create a new Cloud Pub/Sub topic for **Default telemetry topic**.
+    - Click on the **Select a Cloud Pub/Sub topic** Dropdown.
+    - Click on the **Create a topic** option.
+    - Enter a PubSub topic name, such as `gateway-telemetry`.
+8. Do the same for **Device state topic**, under a different PubSub topic named `gateway-state`.
+
+    ![create registry][create-registry]
+
+9. Leave everything else as-is, and click **Create**.
+
+[create-registry]: ./images/create-registry.png 
+
+## Set Up your Gateway
```

Comment:
I'll stop marking the headings, but all should be in sentence case.

Reply:
Changed the casing for the other headings